providers/infrastructure-vsphere/v1.5.0/ytt/overlay.yaml

```diff
@@ -299,6 +303,10 @@ spec:
       resourcePool: #@ data.values.VSPHERE_RESOURCE_POOL
       server: #@ data.values.VSPHERE_SERVER
       template: #@ data.values.VSPHERE_TEMPLATE
+#@ if data.values.VSPHERE_WORKER_HARDWARE_VERSION != "" and data.values.VSPHERE_WORKER_HARDWARE_VERSION != None:
+      #@overlay/match missing_ok=True
+      hardwareVersion: #@ data.values.VSPHERE_WORKER_HARDWARE_VERSION
```

Comment:
Is there any change required for cc based clusters?

Reply:
I have observed some other missing items from the cc VSphereMachineTemplates, such as PCI devices, Ipv6 settings. I was hoping to raise an issue for this, as a catch up for all these missing items, and have a single PR bridge the gap between the two template types.